### DISCUSSION TITLE
Scale DoubleTime score multiplier with the speed applied

### DISCUSSION THREAD

**Discussion by averyunderstars** [ppy/osu#21875] [Ideas]
Currently, DoubleTime gives its 1.12x multiplier only when set to 1.5x speed - not more, not less. It was said that this is temporary, and other people have noticed it as well (#21863), so here's an idea for a future solution.

I think it'd be a good idea to change it based on the speed of the mod - the bigger the increase, the bigger the multiplier, and vice versa - the smaller the speed increase, the smaller the multiplier. _**If my math is correct**_, the multiplier should increase by 0.24x for every tenth of a speed increase, for example:
[Speed] - [Multiplier]
1.01x - 1.0024x
1.10x - 1.024x
1.20x - 1.048x
1.25x - 1.060x
1.50x - 1.12x
1.60x - 1.136x
1.75x  - 1.172x

Now, I am almost certain that my calculations are wrong, so if they are wrong, and if my idea makes sense to you, and you know how to correct them, please do it in the replies :)

**Comment by CaptainBasculin** [comment]
I think this approach makes sense, but instead of a linear increase there should be a curve done to accurately represent the speed mod differences with HalfTime too (where 0.75x speed stands for 0.3x score)

**Reply by averyunderstars** [reply]
Forgot to mention HalfTime whoops
How would the curve look?
